### ISSUE TITLE
Sharing Device web do not give any information, when link is opened at time before Time Range with self signed SSL

### ISSUE THREAD

**Issue by nicrame** [Ylianst/MeshCentral#3502]
When i Share device, and select some future Time Range when it should be active, for example 2022-01-21 15:00 to 2022-01-31 15:00, and I or anyone enter the URL before that time, then page is empty without any message. For example:
![Zrzut ekranu (42)](https://user-images.githubusercontent.com/5872054/150514062-1eb748ec-496e-4458-8638-e56646e769b8.png)
Here browser window (it's same result for Chrome and Firefox):
![Zrzut ekranu (44)](https://user-images.githubusercontent.com/5872054/150514870-5e6046a4-7190-49db-b47b-ed18b490594d.png)
This happening only when SSL certificate is not valid and user must accept it manually.
When certificate is trusted (for example when root server is added inside browser config) then there is correct message.

Also, there is lacking of translation on the shared device webpage (but this is not connected to the problem above).

**Comment by Ylianst** [comment]
I can't replicate this even with a un-trusted SSL cert.

![image](https://user-images.githubusercontent.com/1319013/150572323-2af96df9-fc9e-42b7-8411-d2f063bb9482.png)

Open the browser console and let me know if you see any errors.

**Comment by nicrame** [comment]
I tried again and - in Chrome it worked, but in Firefox i still have same problem - but only in Private/Incognito Window. And only in Firefox that is built with Polish language. I've installed English one and there were no problems (but maybe i should try with clear, separate fresh install too).
But, in the console log it did show some error:
![Zrzut ekranu (514)](https://user-images.githubusercontent.com/5872054/150582644-e816254b-59c2-4305-b4c1-0281c2214355.png)
when i click on the error:
![Zrzut ekranu (515)](https://user-images.githubusercontent.com/5872054/150582716-5f9e02d8-0a2d-4d75-b1c0-86ec33bc2b13.png)
